FusionIIIT/applications/visitor_hostel/models.py

```diff
@@ -125,6 +132,7 @@ class Bill(models.Model):
     meal_bill = models.IntegerField(default=0)
     room_bill = models.IntegerField(default=0)
     payment_status = models.BooleanField(default=False)
+    bill_date = models.DateField()

```

Comment:
use NULL=true , blank=true, for all such models where some entries already exists and you add a new column

Reply:
Done